.circleci/src/workflows/site-factory.yml.tpl

```diff
@@ -1,19 +1,20 @@
-jobs:
-  # Git jobs
-  #
-  # Check validity of git history
-  - lint-git:
-      filters:
-        tags:
-          only: /.*/
-  # Check that circleci configuration is updated
-  - check-configuration:
-      ci_update_options: "${CI-UPDATE-OPTIONS}"
-      filters:
-        branches:
-          ignore: master
-  # Lint bash scripts in /bin
-  - lint-bash:
-      filters:
-        tags:
-          only: /.*/
+site-factory:
```

Comment:
I didn't get how it was broken. It would be interesting to explain what happened in your commit message.

Reply:
Because site-factory is now autogenerate. So if it changes, the job `check_configuration` will failed.
As we do not commit @{site}.yml, now we ignore @site-factory.yml